torch/_dynamo/trace_rules.py

```diff
@@ -93,6 +94,29 @@
     "torch.nn.Parameter": SkipFilesVariable,
     "torch._nested_tensor_from_mask": SkipFilesVariable,
     "torch._nested_from_padded": SkipFilesVariable,
+    # functorch
+    "torch._functorch.vmap._check_int_or_none": UserFunctionVariable,
+    "torch._functorch.vmap._check_out_dims_is_int_or_int_pytree": UserFunctionVariable,
+    "torch._functorch.vmap._check_randomness_arg": UserFunctionVariable,
+    "torch._functorch.vmap._chunked_vmap": UserFunctionVariable,
+    "torch._functorch.vmap._concat_chunked_outputs": UserFunctionVariable,
+    "torch._functorch.vmap._create_batched_inputs": UserFunctionVariable,
+    "torch._functorch.vmap._flat_vmap": UserFunctionVariable,
+    "torch._functorch.vmap._flatten_chunks_output": UserFunctionVariable,
+    "torch._functorch.vmap._get_chunked_inputs": UserFunctionVariable,
+    "torch._functorch.vmap._get_name": UserFunctionVariable,
+    "torch._functorch.vmap._maybe_remove_batch_dim": UserFunctionVariable,
+    "torch._functorch.vmap._num_outputs": UserFunctionVariable,
+    "torch._functorch.vmap._process_batched_inputs": UserFunctionVariable,
+    "torch._functorch.vmap._unwrap_batched": UserFunctionVariable,
+    "torch._functorch.vmap._validate_and_get_batch_size": UserFunctionVariable,
+    "torch._functorch.vmap.doesnt_support_saved_tensors_hooks": UserFunctionVariable,
+    "torch._functorch.vmap.get_chunk_sizes": UserFunctionVariable,
+    # "torch._functorch.vmap.lazy_load_decompositions": UserFunctionVariable,
+    "torch._functorch.vmap.restore_vmap": UserFunctionVariable,
+    "torch._functorch.vmap.unwrap_batched": UserFunctionVariable,
+    "torch._functorch.vmap.vmap_impl": UserFunctionVariable,
+    "torch._functorch.vmap.wrap_batched": UserFunctionVariable,
```

Comment:
@yanboliang is there a succinct way to express that we want everything in "torch._functorch.vmap" to be inlined?

Reply:
Yes, two steps:
* Add ```torch._functorch.vmap``` to https://github.com/pytorch/pytorch/blob/626b7dc847bff19c70b200fa2228bf2b8f4df753/torch/_dynamo/skipfiles.py#L197
* Remove majority ```torch._functorch.vmap``` functions from this list.
  - You have to keep the function that dynamo have implementation in this list, e.g., ```torch._functorch.vmap.vmap_impl```.

Reply:
I think we already did that, but for some reason it didn't work?

Reply:
Oh, then you have to do the following two steps:
* Remove these functions from the trace_rules.py list, because it takes precedence over the ```skipfiles.MOD_INLINELIST``` check.
* Wait my next PR to remove ```is_allowed```, which I plan to publish in 1-2 days.

Then these functions would be inlined.